src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -809,6 +820,68 @@ public void snapshotTwoColocatedNonEmptyAndNonColocatedEmptyThenStream(boolean c
         assertEquals(401, recordsForNonColocated.size());
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void snapshotTwoColocatedNonEmptyAndNonColocatedEmptyThenStreamWithConsistentSnapshot(boolean colocation) throws Exception {
+        /* The objective of this test is to verify that we are able to consume all snapshot records on a
```

Comment:
Multi line comments follow this pattern:
```java
/*
 * Starting line of the comment would be here
 * and it will continue. You can end it on the next line as shown.
 */
```